File: crm-backend/src/main/java/edu/yavirac/crmbackend/feature/acesor/Acesor.java
package edu.yavirac.crmbackend.feature.acesor;


import java.sql.Date;
import java.util.HashSet;
import java.util.Set;

import org.springframework.data.annotation.Id;
import org.springframework.data.relational.core.mapping.Column;
import org.springframework.data.relational.core.mapping.MappedCollection;
import org.springframework.data.relational.core.mapping.Table;


import lombok.Data;

@Data
@Table("acesores")
public class Acesor {
    @Id
    @Column("acesor_id")
    private long acesorId;
    private String nombre;
    private String telefono;
    private String email;
    private Long personId;
    private boolean enabled;
    @MappedCollection(idColumn = "acesor_id")
    private Set<AcesorClient> clients = new HashSet<>();
    private Date created;
    private Date updated;
}

File: crm-frontend/src/app/feature/acesor/acesor.ts
import { AcesorClient } from "./AcesorClient";

export interface Acesor
 {
   acesorId:number,
   nombre: string,
   telefono: string,
   email: string,
   personId: number,
   enabled: boolean,
   //clients:AcesorClient[],
   created:Date,
   updated:Date
   clients:AcesorClient[],

}
